.envrc

```diff
@@ -0,0 +1,195 @@
+# This is the .envrc for Sentry, for use with direnv.
+# It's responsible for enforcing a standard dev environment by checking as much state as possible, and either performing
+# initialization (e.g. activating the venv) or giving recommendations on how to reach the desired state.
+# It also sets useful environment variables.
+# If you'd like to override or do any custom things, this .envrc will source a .envrc-overrides file at the end.
```

Comment:
Total bike shed, but why not wrap this to 80 col

Reply:
🙄